README.md

```diff
@@ -15,6 +15,31 @@ To install this module in Julia, use
 ] add https://github.com/oscar-system/GAP.jl
 ```
 
+## Basic usage
+
+After
```

Comment:
```suggestion
After entering the following in Julia,
```
This way it is clear that we are talking about Julia code, and it applies both to interactive and non-interactive usage.